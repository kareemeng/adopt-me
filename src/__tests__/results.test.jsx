import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import Results from "../Results";

test("renders without error", async () => {
  const { asFragment } = render(<Results pets={[]} />);
  expect(asFragment()).toMatchSnapshot();
});
