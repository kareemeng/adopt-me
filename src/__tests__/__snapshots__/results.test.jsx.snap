// Vitest Snapshot v1

exports[`renders without error 1`] = `
<DocumentFragment>
  <div
    class="search"
  >
    <h1>
      No Pets Found
    </h1>
  </div>
</DocumentFragment>
`;
